cassandra/pom.xml

```diff
@@ -40,7 +40,7 @@
         <lz4.version>1.2.0</lz4.version>
         <scala.version>2.11.7</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
-        <commons-lang.version>3.3.2</commons-lang.version>
+        <commons-lang.version>3.4</commons-lang.version>
```

Comment:
Yes, org.seleniumhq.selenium - 2.48.2 requires commons-exec:commons-exec - 1.3 which in turn will require commons-lang3 - 3.4. Hence, had to upgrade all.